istioctl/pkg/wait/wait_test.go

```diff
@@ -77,7 +77,7 @@ func TestWaitCmd(t *testing.T) {
 		},
 		{
 			execClientConfig: cannedResponseMap,
-			args:             strings.Split("--generation=1 VirtualService foo.default --proxy not-proxy", " "),
+			args:             strings.Split("--generation=1 --timeout 20ms VirtualService foo.default --proxy not-proxy", " "),
```

Comment:
I do think similarly as @bleggett this case should be same as others

Reply:
Done. I refactored a bit to ensure we always check the error message (so we check the right error, not that it timed out)